recipes/libiconv/all/conanfile.py

```diff
@@ -85,6 +85,8 @@ def _build_context(self):
                     yield
 
     def _configure_autotools(self):
+        if self.settings.build_type == "RelWithDebInfo":
+           self.settings.build_type = "Release"
```

Comment:
overriding settings should not be done. Maybe you need to adjust the autotools flags instead

Reply:
well, I don't really know could I do that. Anyway, in my opinion (and considering that RelWithDebInfo is a cmake only thing not autotools' but still it is binary compatible with Release builds), it should be Conan who manages this situation. I.e. if autotools has no concept of release with debug info, then a RelWithDebInfo conan build should fallback to Release for a dependency that uses Autotools'. It doesn't seem to be the case, so this is what I came up with. It works for me and I thought it would be helpful for others. If you know how to make the adjustment you mention, then please let me know and I will. Otherwise, this is as far as I can get and I will understand if you reject this.

Reply:
there is no standard for such configurations. `RelWithDebInfo` is CMake exclusive thing. other build system may define their own build types, e.g. profile, sanitize, code coverage, etc.
the more or less de-facto standards are:
- Debug - optimization is disabled (sometimes also DEBUG or _DEBUG are defined)
- Release - optimization is enabled (but level might be any), defined _NDEBUG
others, are even less standardized and formalized. Autotools, which predates CMake, obviously has no such concept. 
I'd probably just alias RelWithDebInfo to Release. they are more equivalent, except debug information, but such information is usually provided in its own file (.debug, .pdb or .dwarf, it depends on platform/linker/binary format/etc), so you have a choice to distribute or not distribute it. but in conan we never include debug information files into the package (for reasons), so from the conan perspective, it's absolutely the same.